docs/chainlink-keepers/cron-contracts.md

```diff
@@ -0,0 +1,626 @@
+
+![Remix Transaction Hash](/images/keepers/cron-2.png)
+
+## Interacting with your Job Scheduler
+
+To interact with your Job Scheduler contract, create a new file called `CronUpkeep.abi` in Remix and paste the [ABI](#job-scheduler-aka-cronupkeep-abi) for `CronUpkeep` contract from the [Resources](#resources) section below. Select the **DEPLOY & RUN TRANSACTIONS** icon on the left and paste the address of your Job Scheduler contract in the **At Address** box, and then select the **At Address** button. Please accept the prompts after which the contract interface will open in the bottom left of Remix.
+
+![Remix Contract Interface](/images/keepers/cron-3.png)
+
+## Scheduling a Job
+
+To schedule a new job, use the function `createCronJobFromEncodedSpec`. This function has three inputs: 
+
++ The target address of the smart contract you want to automate
++ The [function signature](#function-signature), or the encoding of the function, you want to call 
++ The [encodedCronSpec](#encodedcronspec) encoding of your specified time interval.
+
+Please see the [Encodings](#encodings) section below to get the [function signature](#function-signature) and the [encodedCronSpec](#encodedcronspec) for your job. 
+
+We will now paste, minus <> brackets, 
+```json
+<your target address>, <your function specification>, <your encodedCronSpec>
+```
+into the into the createCronJobFromEncodedSpec function in Remix and execute it. Once executed, you can call the getActiveCronJobIDs to see if your job has been registered. You can execute the getCronJob function to see the details of your job played back. You can add multiple jobs on this contract in this fashion, and also use the other functions in your Job Scheduler to delete jobs or pause and unpause the Scheduler.
+
+You still have to register your Job Scheduler with the Chainlink Keepers service to have it monitored and executed.
+
+## Setting up Chainlink Keepers
+
+Since our deployed contract is already [Keeper-compatible](../compatible-contracts), we can immediately register it with Keepers following these [steps](../register-upkeep). Be sure to provide the address of your Job Scheduler contract in **Upkeep address**. For Gas limit you should specify the upper limit of Gas your target function will use.
+
+Once you have registered your contract, you can view your Upkeep. The *Upkeep History* will display **Perform Upkeep** for all registered jobs on your Job Scheduler contract. To ensure Chainlink Keepers monitors your Job Scheduler, please ensure you fund your Upkeep.
+
+# Summary
+
+In this tutorial you used the Chainlink Keepers Job Scheduler to automate the execution of your smart contract functions at specific times. 
+
+# Encodings
+
+By using encodings you save the repeated gas costs of having your contract calculate them during execution.
```

Comment:
```suggestion
Encodings allow you to save repeated gas costs of having your contract calculate them during execution.
```

Reply:
Is this sentence necessary to include here? It may be better to include when encodings are first introduced in the walkthrough